week03/assignment03_test.go

```diff
@@ -0,0 +1,223 @@
+package week03
+
+import (
+	"errors"
+	"testing"
+)
+
+// ✅ Define a Movie struct and export two public fields; Length of type int and Name of type string. You must NOT export any more public fields on Movie. You are allowed, however, to use non-exported, private fields as needed.
+// ✅ Define a method on the pointer receiver for Movie named Rate. Rate should take a float32 rating and return an error. Calling Rate should track this rating. If the number of plays is 0 return the following error: fmt.Errorf()
+func TestMovieRate(t *testing.T) {
+	movie := &Movie{}
+
+	t.Run("NoPlaysReturnError", func(t *testing.T) {
+		movie.plays = 0
+		err := movie.Rate(0.5)
+
+		if !errors.Is(err, ErrRatingNoPlays) {
+			t.Errorf("Expected error to be %v got %v", ErrRatingNoPlays, err)
```

Comment:
Error messages in Go should not start with an uppercase letter.

Reply:
Thanks!